ClosedXML/Excel/PivotTables/PivotStyleFormats/XLPivotFieldStyleFormats.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Keep this file CodeMaid organised and cleaned
+
+namespace ClosedXML.Excel
+{
+    internal class XLPivotFieldStyleFormats : IXLPivotFieldStyleFormats
+    {
+        private IXLPivotValueStyleFormat dataValuesFormat;
+        private IXLPivotStyleFormat headerFormat;
+        private IXLPivotStyleFormat labelFormat;
+        private IXLPivotStyleFormat subtotalFormat;
+
+        public XLPivotFieldStyleFormats(IXLPivotField field)
+        {
+            this.PivotField = field;
+        }
+
+        public IXLPivotField PivotField { get; }
+
+        #region IXLPivotFieldStyleFormats
+
+        public IXLPivotValueStyleFormat DataValuesFormat
+        {
+            get { return dataValuesFormat ?? (dataValuesFormat = new XLPivotValueStyleFormat(PivotField) { AppliesTo = XLPivotStyleFormatElement.Data }); }
```

Comment:
A bit too much for one line. Better use for `DataValuesFormat` and for `Header` the same pattern as for `Label` and `Subtotal`.